scan_fmt_scxml/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "scan_scxml"
```

Comment:
The name conflicts with the other `scan_scxml`. I suggest renaming your package and library crate as `scan_fmt_scxml`.